_posts/minutes/2016-10-07-minutes.markdown

```diff
@@ -1,4 +1,4 @@
-[LISTEN TO THIS MEETING](https://fccdl.in/iPvoj2r1R)
+&nbsp;&nbsp;&nbsp;Listen to this meeting here: [here](https://fccdl.in/iPvoj2r1R)

```

Comment:
Well sir, I would like to know how I can display it better. 😝 . I thought it was a step up from what it was before. 😆